chart/compass/charts/gateway/templates/virtualservice.yaml

```diff
@@ -72,6 +72,7 @@ spec:
           - "authorization"
           - "content-type"
           - "tenant"
+          - "client_user"
```

Comment:
Don't forget to not leave the value hard coded here and in the other virtualservice file as well